client/utilities/hooks/useAnalytics.ts

```diff
@@ -2,12 +2,6 @@ import { loadScript } from '@guardian/libs';
 import * as Sentry from '@sentry/browser';
 import { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
-import parse from 'url-parse';
```

Comment:
does anywhere else use `url-parse` ? maybe we can get rid of it

Reply:
great suggestion. I have removed it!